### PR TITLE
Suthesh/hide-real-account-derivx

### PR DESCRIPTION
**Pull request by suthesan-deriv**
Changes:

-   hide real account derivx

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/derivx/_hero.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import { size } from 'themes/device'
+import DerivXBG from 'images/svg/deriv-x/triangle-up.svg'
+import DerivXBG2 from 'images/svg/deriv-x/triangle-down.svg'
+import deriv_x_logo from 'images/svg/deriv-x/derivx-logo.svg'
+import { isBrowser } from 'common/utility'
+
+const Hero = () => {
+    const [is_mobile, setMobile] = useState(false)
+
+    const handleResizeWindow = useCallback(() => {
```
Comment (ShuvoHabib) [inline src/pages/derivx/_hero.js:13]: Can we create an Utility function for this?

File: src/pages/derivx/_start-derivx.js [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+const StyledText = styled(Text)`
+    @media ${device.mobileL} {
+        font-size: 16px;
+    }
+`
+
+const StartDerivX = () => {
+    const [is_mobile, setMobile] = useState(false)
+    const handleResizeWindow = useCallback(() => {
+        setMobile(isBrowser() ? window.screen.width <= size.tablet : false)
```
Comment (ShuvoHabib) [inline src/pages/derivx/_start-derivx.js:184]: We can create an Utility function

File: src/pages/dmt5/_start-trader.js [hunk 3]
```diff
@@ -239,71 +239,71 @@ const StartTrader = () => {
...
-                            <ImageWrapper>
-                                <QueryImage
-                                    data={data[is_mobile ? 'real_step4_mobile' : 'real_step4']}
-                                    alt="real_step4"
-                                />
-                            </ImageWrapper>
-                        </SideTab.Panel>
-                    </SideTab>
-                )}
+                        <SideTab parent_tab={tab}>
```
Comment (ShuvoHabib) [inline src/pages/dmt5/_start-trader.js:242]: Looks like extra spacing.

File: src/pages/derivx/_what-is-derivx.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    }
+`
+
+const WhatIsDeriv = () => {
+    return (
+        <StyledSectionContainer>
+            <Container>
+                <Flex width="792px" fd="column" ai="center" jc="center">
+                    <Header type="page-title" align="center">
+                        <Localize translate_text="What is Deriv X" />
```
Comment (ShuvoHabib) [inline src/pages/derivx/_what-is-derivx.js:22]: Is there a question mark here ?

File: src/pages/derivx/_start-derivx.js [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+    const [tab, setTab] = useState('demo')
+
+    const onTabClick = (tab) => {
+        setTab(tab)
+    }
+
+    return (
+        <Section>
+            <StyledHeader align="center" mb="4rem" as="h2" type="page-title">
+                {localize('How to get started with a Deriv X account')}
```
Comment (ShuvoHabib) [inline src/pages/derivx/_start-derivx.js:206]: Is there a question mark here?

File: src/pages/derivx/_why-trade-derivx.js [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+        <div>
+            <SectionContainer>
+                <Container fd='column'>
+                    <StyledHeader type='page-title' align="center">
+                        {localize('Why trade with Deriv X')}
+                    </StyledHeader>
+                    <Flex tablet_direction='column' tablet_ai='center' mt="40px">
+                        {card_data.map((index) => {
+                            return (
+                                <Card key={index}>
```
Comment (ShuvoHabib) [inline src/pages/derivx/_why-trade-derivx.js:109]: Can we use  a single key for 'key' not the full object?

**Review by ShuvoHabib — COMMENTED** [review]
A few minor changes

File: src/pages/derivx/_what-is-derivx.js (hunk 4 above)
Comment (suthesan-deriv) [reply src/pages/derivx/_what-is-derivx.js:22]: According to design there's no question mark here

File: src/pages/derivx/_start-derivx.js (hunk 5 above)
Comment (suthesan-deriv) [reply src/pages/derivx/_start-derivx.js:206]: According to design there's no question mark here

File: src/pages/derivx/_start-derivx.js (hunk 2 above)
Comment (suthesan-deriv) [reply src/pages/derivx/_start-derivx.js:184]: I think we can handle this on a separate card, cause I see a lot of this being used elsewhere

**Review by sean-binary — APPROVED** [review]

**Pull request merged**